app/backend/src/tests/test_interceptors.py

```diff
@@ -223,7 +223,7 @@ def TestRpc(request, context):
         request_type=auth_pb2.SignupAccount,
         response_type=auth_pb2.AuthReq,
     ) as call_rpc:
-        with pytest.raises(Exception):
+        with pytest.raises(Exception, match="Some error message"):
```

Comment:
What does this line do?

Reply:
It checks that the exception message matches the given regex. This is to prevent the linting rule B017 https://docs.astral.sh/ruff/rules/assert-raises-exception/